applications/jupyter/main.tf

```diff
@@ -77,6 +77,16 @@ module "namespace" {
   create_namespace = true
 }
 
+# IAP Section: Enabled the IAP service
+resource "google_project_service" "project_service" {
```

Comment:
can this be moved into iap module

Reply:
Discussed offline, we will do another test later. From my option this is project level, should be enabled once. and Umesh thinks it is good for code maintain and generality of IAP module, also the terraform will auto correct it.